Assets/Plugins/QuestManager/Scripts/Runtime/QuestView.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace GameLabGraz.QuestManager
```

Comment:
Can we move the view scripts into a separate folder?
and use a different namespace for it e.g. GameLabGraz.QuestManager.View with a separate assembly definition file